### ISSUE TITLE
BUG: code that crashes the simulator (Javascript Bridge)

### ISSUE THREAD

**Issue by jsfan3** [codenameone/CodenameOne#2200]
The following code crashes the simulator. The Project Libs are updated on 1st September 2017.

I attach the java code, the html code, the NetBeans log. I also reported this bug in this StackOverlow page: https://stackoverflow.com/questions/45795939/codename-one-browsercomponent-use-the-back-button-to-go-back-in-history

Java code:

```
    public void start() {
        if (current != null) {
            current.show();
            return;
        }
        Form hi = new Form("MyBrowser", new BorderLayout());

        // Create the BrowserComponent
        BrowserComponent browser = new BrowserComponent();

        try {
            // Load a web page placed in the "html" package
            browser.setURLHierarchy("/index.html");
            Log.p("setURLHierarchy executed");
        } catch (IOException ex) {
            Log.e(ex);
        }

        hi.add(BorderLayout.CENTER, browser);

        // Javascript Bridge
        JavascriptContext context = new JavascriptContext(browser);
        JSObject document = (JSObject) context.get("document");

        Command backCommand = new Command("BackButton") {
            @Override
            public void actionPerformed(ActionEvent evt) {
                // browser.back();
                document.call("writeln", new Object[]{"Hello world"});
            }
        };

        hi.getToolbar().setBackCommand(backCommand);
        hi.show();
    }
```

index.html:
```
<!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
    </head>
    <body>
        <a href="page1.html">Page 1</a><br />
        <a href="page2.html">Page 2</a><br />
        <a href="page3.html">Page 3</a><br />
    </body>
</html>
```

NetBeans output log:
```
ant -f "/home/francesco/NetBeansProjects/MySmallTests/Bugs/Bug 1" -Dnb.internal.action.name=run run
No GUI Entries available
init:
Deleting: /home/francesco/NetBeansProjects/MySmallTests/Bugs/Bug 1/build/built-jar.properties
deps-jar:
Updating property file: /home/francesco/NetBeansProjects/MySmallTests/Bugs/Bug 1/build/built-jar.properties
Compile is forcing compliance to the supported API's/features for maximum device compatibility. This allows smaller
        code size and wider device support
compile:
run:
[EDT] 0:0:1,94 - Codename One revisions: 7019176aba16d00a894250876ae5fcdc9634e23c

[EDT] 0:0:1,110 - setURLHierarchy executed
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f23d16b64d1, pid=11317, tid=0x00007f23e0478700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_144-b01) (build 1.8.0_144-b01)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.144-b01 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libjfxwebkit.so+0x11334d1]  checkJSPeer(long, int, OpaqueJSValue*&, OpaqueJSContext const*&)+0x41
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/francesco/NetBeansProjects/MySmallTests/Bugs/Bug 1/hs_err_pid11317.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Java Result: 134
BUILD SUCCESSFUL (total time: 4 seconds)
```

**Comment by codenameone** [comment]
This is a duplicate of #2202 with the same answer

**Comment by jsfan3** [comment]
The bug is published with all details and steps to reproduce:
http://bugs.java.com/bugdatabase/view_bug.do?bug_id=JDK-8187568

However, I've spent a lot of time to report the bug and to workaround it, but who verified it commented so: "[...] but could not reproduce the issue". So... no fix :(

**Comment by codenameone** [comment]
Check out how their configuration might differ from yours in subtle ways.
